src/PosthogAnalytics.ts

```diff
@@ -367,7 +387,25 @@ export class PosthogAnalytics {
         this.registerSuperProperties(this.platformSuperProperties);
     }
 
+    private updateCryptoSuperProperty(): void {
+        if (!this.enabled || this.anonymity === Anonymity.Disabled) return;
+        // Update super property for cryptoSDK in posthog.
+        // This property will be subsequently passed in every event.
+        const value = this.getCryptoSDKPropertyValue();
+        this.registerSuperProperties({ cryptoSDK: value });
```

Comment:
Don't these clobber between sessions of the same account?

Reply:
No I tested (same users with a rust and a legacy "device")
<img width="1361" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/9841565/585a13e1-3fcd-466e-ab5c-f32c5752ae3e">


As per doc 
> However, please note that this does not store properties against the User, only against their events. 

It's technically stored in a cookie:
> Setting Super Properties creates a cookie on the client with the respective properties and their values.

There is a thing maybe annoying: if you switch to rust stack and we do PageView per unique user, the same user will report in both Legacy and Rust for some time. However for web insights we can use per [Unique Session](https://posthog.com/docs/data/sessions) that could be better for us

Reply:
> Setting Super Properties creates a cookie on the client with the respective properties and their values.

Then we can't use it without updating the [cookie policy](https://static.element.io/legal/cookie-policy.pdf).

Reply:
> > Setting Super Properties creates a cookie on the client with the respective properties and their values.
> 
> Then we can't use it without updating the [cookie policy](https://static.element.io/legal/cookie-policy.pdf).

We are already using superProperties for `appVersion` and `appPlatform`. So the cookie policy is already outdated?